app/scripts/webgl/html3d/context.js

```diff
@@ -67,22 +67,22 @@ export default class GGContext {
         return;
       }
 
-      const container = scene.container;
+      const { container } = scene;
 
       const rect = container.getBoundingClientRect();
 
       if (
-        rect.bottom < 0 ||
-        rect.top > this.canvas.clientHeight ||
-        rect.right < 0 ||
-        rect.left > this.canvas.clientWidth
+        rect.bottom < 0
+        || rect.top > this.canvas.clientHeight
+        || rect.right < 0
+        || rect.left > this.canvas.clientWidth
       ) {
         return;
       }
 
       const width = rect.right - rect.left;
       const height = rect.bottom - rect.top;
-      const left = rect.left;
+      const { left } = rect;
```

Comment:
👌 Z tohto maju ostatni devs dobru zabavu, ked najdu destruct v martinus JS kode :D Ale ja to mam rad :)

Reply:
![b9c4b2ec9a9b62f54ae5672a558608c1](https://user-images.githubusercontent.com/2431529/109005166-f1032d80-76a9-11eb-8494-d148ac2d7695.gif)